docs/integrations/ios-specific-workflow-steps/swift-lint.md

```diff
@@ -0,0 +1,77 @@
+:::
+:::caution
+If you are using **Swift Package Manager(SPM)**, do not use this step. SPM packages will be compiled in other steps that work with **Xcode**, such as [**Xcodebuild for Devices**](https://docs.appcircle.io/workflows/ios-specific-workflow-steps#xcodebuild-for-devices-archive--export)
+
+If you have SPM in your project and you are using SwiftLint component in workflow. The Linter component will give an error because it cannot find the required dependencies.
+:::
+
+### Step Details
+
+You can customize your SwiftLint step with the options in the component. You can also customize your SwiftLint step with the options in the component. You can also find detailed descriptions of all variables in the table below.
+
+- With the Range option, you can run SwiftLint on your entire project or only on changes in PR.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2613-lint_range.png' />
+
+- You can change the report type with the Reporter format option. This option supports extensions such as **`html`**, **`json`**, **`junit`** etc.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2613-lint_format.png' />
+
+- If there is a failure in the running lint, you can break the pipeline with the **Strict** option, if you want the logs to be simpler, you can make the report file simpler with the **Quiet Mode** feature.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2613-lint_strict.png' />
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2613-lint_quiet.png' />
+
+### Input Variables
+
+You can find all the parameters required for this step in the table below with their descriptions in detail.
+
+:::note
+Inputs marked with an asterisk (*) denote required fields.
+:::
+
+| Variable Name                 | Description                                    |
+|-------------------------------|------------------------------------------------|
+| `$AC_REPOSITORY_DIR`*         | Specifies the cloned repository directory. |
+| `$AC_LINT_PATH`               | Specifies the path to lint. |
+| `$AC_LINT_RANGE`              | Take the git diff and only lint those files. Default is **all** |
+| `$AC_LINT_CONFIG`             | Specifies linting configuration file. For example : /.swiftlint.yml |
+| `$AC_LINT_REPORTER`           | The custom reporter format to use. Default is **xcode** |
+| `$AC_LINT_STRICT`             | Use strict mode for linting. This will fail the build if linting fails. Default is NO |
+| `AC_LINT_QUIET`               | Don't print status logs like 'Linting ' & 'Done linting' |
+
+### Output Variables
+
+| Variable Name                 | Description                                    |
+|-------------------------------|------------------------------------------------|
+| `$AC_LINT_OUTPUT_PATH`*       | The path of the SwfitLint results output file. |
```

Comment:
@boztopuz  Can you remove `*` from the output table, because the output has not `required` field.